src/BinaryChart.js

```diff
@@ -41,11 +41,13 @@ export default class BinaryChart extends Component {
         const config = initChart(newProps || this.props);
         config.chart.renderTo = this.refs.chart;
         this.chart = new Highcharts.StockChart(config);
+        
     }
 
     componentDidMount() {
         this.createChart();
         updateChart(this.chart, { ticks: [] }, this.props);
+        this.chart.rangeSelector.clickButton(0, 0, true);
```

Comment:
Can this not be set by passing prop to chart on initialize?
This way, it will be a chart repaint, then another chart repaint.